nft-simple/src/internal.rs

```diff
@@ -153,3 +156,33 @@ impl Contract {
         (owner_id, approved_account_ids)
     }
 }
+
+pub(crate) fn convert_token_to_ext_object(token: Token) -> TokenReturnObject {
+    let mut json_map: HashMap<AccountId, U64> = HashMap::new();
+    for hm in token.approved_account_ids.clone() {
+        json_map.insert(hm.0, U64::from(hm.1));
+    }
```

Comment:
I don't think we need approval_id to ever be U64. We can't reach it possibly

Reply:
Do you mean we should have it be smaller? Or use `u64` instead? Not sure I'm reading the comment correctly.

Reply:
I think approval can be just `u64`, but we can always just use `U64`.

Reply:
changed to use `U64` everywhere